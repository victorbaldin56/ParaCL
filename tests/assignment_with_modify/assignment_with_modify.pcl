x = ?;
x += x;
print x;
x -= 5;
print x;
x *= 20;
print x;
x /= 6;
print x;
x %= 3;
print x;
x |= 2;
print x;
x &= 1;
print x;
x <<= 10;
print x;
x >>= 5;
print x;
