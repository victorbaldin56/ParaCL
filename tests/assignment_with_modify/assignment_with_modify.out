2000
1995
39900
6650
2
2
0
0
0
